parquet-column/src/main/java/org/apache/parquet/column/impl/ColumnWriterBase.java

```diff
@@ -389,7 +400,14 @@ void writePage() {
     this.rowsWrittenSoFar += pageRowCount;
     if (DEBUG) LOG.debug("write page");
     try {
-      writePage(pageRowCount, valueCount, statistics, repetitionLevelColumn, definitionLevelColumn, dataColumn);
+      writePage(
+          pageRowCount,
+          valueCount,
+          statistics,
+          sizeStatisticsBuilder.build(),
```

Comment:
Can we have parity of line 406 and 407? You can use a varaiible in line 407

Reply:
Fixed